### ISSUE TITLE
Add additional clarification to media overlays purpose and scope

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#847]
The media overlays purpose and scope section doesn't speak to its accessibility benefits.

Suggest we enhance it by adding the following second paragraph:

"The text and audio synchronization enabled by Media Overlays provides enhanced accessibility for any user who has difficulty following the text of a traditional book. They can also provide a continuous listening experience for readers who are unable to read the text for any reason, something that traditional audio embedding techniques cannot offer. They are even useful for purposes not traditionally considered accessibility concerns (e.g., for language learning or reading of commercial audio books)."

**Comment by iherman** [comment]
+1

**Comment by mattgarrish** [comment]
Proposed Solution

Add the new prose in the first comment.
